src/Uno.UI/UI/Xaml/Controls/Image/Image.iOS.cs

```diff
@@ -261,6 +261,19 @@ private void UpdateContentMode(Stretch stretch)
 			}
 		}
 
+		public override void LayoutSubviews()
+		{
+			try
+			{
+				_layouter.Arrange(Bounds);
```

Comment:
What if Measure was never called ?

Reply:
What do you mean? In that case the image will have a size `0, 0`.